packages/toolkit/src/lambda/commands/uploadLambda.ts

```diff
@@ -209,16 +204,15 @@ export interface UploadLambdaWizardState {
 }
 
 export class UploadLambdaWizard extends Wizard<UploadLambdaWizardState> {
-    constructor(lambda?: LambdaFunction, invokePath?: vscode.Uri) {
-        super({ initState: { lambda } })
+    public override async init(): Promise<this> {
```

Comment:
Since `fsCommon.existsFile` is async, I added `Wizard.init()` so that wizards can have async "constructors". This is something we've needed for awhile.